packages/fxa-shared/package.json

```diff
@@ -4,13 +4,11 @@
   "description": "Shared module for FxA repositories",
   "main": "dist/index.js",
   "scripts": {
-    "postinstall": "npm run build",
```

Comment:
Will this end up being different than `npm run build`?

Reply:
I'd like for this `postinstall` to go away with a little tsconfig magic in the future